StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsOAuthPrepane.swift

```diff
@@ -12,6 +12,7 @@ struct FinancialConnectionsOAuthPrepane: Decodable {
 
     let institutionIcon: FinancialConnectionsImage?
     let title: String
+    let subtitle: String?
```

Comment:
Seems this is only used in swiftUI preview but not in the actual code?

Reply:
I believe its used here:

https://github.com/stripe/stripe-ios/pull/3158/files/eea2be8ad448e2affa9bfdfd74c052cfdea07132#diff-402ae56328287b14bdf53304d97ad7f7cc65499a90b5931e8f170e43c122e0ddR38